lib/browser/property-descriptor.ts

```diff
@@ -66,8 +66,10 @@ function patchViaCapturingAllTheEvents() {
     const property = eventNames[i];
     const onproperty = 'on' + property;
     document.addEventListener(property, function (event) {
-      let elt = <Node>event.target, bound;
-      const source = elt.constructor['name'] + '.' + onproperty;
+      let elt = <Node>event.target, bound, source;
+      if (elt) {
+        source = elt.constructor['name'] + '.' + onproperty;
+      }
```

Comment:
could we add an else branch which would at assume `"unknown"` constructor name and then add the `onproperty`? It sucks that the fix now makes source `undefined`, which is hard for debugging.